src/routes/safe/components/Apps/utils.ts

```diff
@@ -8,10 +8,18 @@ import { SafeAppAccessPolicyTypes } from '@gnosis.pm/safe-react-gateway-sdk'
 
 import { SafeApp } from './types'
 
+type AppManifestIcon = {
+  src: string
+  sizes: string
+  type: string
+  purpose: string
+}
+
 export interface AppManifest {
   name: string
-  iconPath: string
-  description: string
+  iconPath?: string
+  description?: string
```

Comment:
> We are trying to follow the standard as much as we can so if no description is provided no description is showed. Do you think is a bad idea and we would continue to make this mandatory?

I would continue making `description` mandatory because we have our own requirements for content in our app